specifyweb/specify/tree_views.py

```diff
@@ -359,6 +360,65 @@ def tree_rank_item_count(request, tree, rankid):
     count = tree_rank_count(tree, rank.id)
     return HttpResponse(toJson(count), content_type='application/json')
 
+TAXON_TREES = ["Mineral", "Rock", "Meteorite", "Fossil"]
+TAXON_RANKS = ["One", "Two", "Three"]
+
+# TODO: Add openapi schema for this endpoint
+@tree_mutation
+def add_geo_default_trees(request):
+    for tree in TAXON_TREES:
+        if Taxontreedef.objects.filter(name=tree).exists():
+            continue
+        tree_name = f"{tree} Taxon"
+        ttd = Taxontreedef.objects.create(name=tree_name)
+        ttd.save()
+        rank_id = 0
+        ttdi = None
+        for rank in TAXON_RANKS:
+            name = f"{tree} {rank}"
+            rank_id += 10
+            ttdi = Taxontreedefitem.objects.create(
+                name=name,
+                title=name,
+                rankid=rank_id,
+                parent=ttdi,
+                treedef=ttd,
+            )
+            ttdi.save()
+
+# TODO: Add openapi schema for this endpoint
+@tree_mutation
+def remove_geo_default_trees(request):
+    for tree in reversed(TAXON_TREES):
```

Comment:
this is useless. If removing this reversed breaks things, you've a worse bug somewhere else